STATIC_CODE_CHECKS.rst

```diff
@@ -171,7 +171,8 @@ require Breeze Docker image to be built locally.
 +-----------------------------------------------------------+--------------------------------------------------------------+---------+
 | check-decorated-operator-implements-custom-name           | Check @task decorator implements custom_operator_name        |         |
 +-----------------------------------------------------------+--------------------------------------------------------------+---------+
-| check-deferrable-default-value                            | Check default value of deferrable attribute                  |         |
+| check-deferrable-default-value                            | * Check default value of deferrable attribute                |         |
+|                                                           | * Check default value of deferrable attribute                |         |
```

Comment:
This seems out of place

Reply:
Yep, I just found out I forget to change the pre-commit name. I've updated it. Thanks!